connect/runtime/src/test/java/org/apache/kafka/connect/runtime/isolation/DelegatingClassLoaderTest.java

```diff
@@ -39,7 +40,9 @@ public void testLoadingUnloadedPluginClass() {
                 Collections.emptyList(),
                 DelegatingClassLoader.class.getClassLoader()
         );
-        classLoader.initLoaders();
+        Set<PluginSource> classLoaders = classLoader.sources();
+        PluginScanResult scanResult = new ReflectionScanner().discoverPlugins(classLoaders);
+        classLoader.installDiscoveredPlugins(scanResult);
```

Comment:
These lines are repeated a few times throughout this test suite; worth pulling out into a reusable method?

```java
private void initLoaders(DelegatingClassLoader classLoader) {
    Set<PluginSource> classLoaders = classLoader.sources();
    PluginScanResult scanResult = new ReflectionScanner().discoverPlugins(classLoaders);
    classLoader.installDiscoveredPlugins(scanResult);
}
```